docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,177 @@
+- `Info per level`, which includes the timestamp and block hash of the preceding block
+- `End of level`, which indicates the end of the block
+
+Smart Rollup nodes can use these internal messages to know when blocks begin and end.
 
 ### Reveal data channel
 
-The reveal data channel is a communication interface that allows the
-rollup to request data from sources that are external to the inbox and
-can be unknown to layer 1. The rollup node has the responsibility to
-answer the rollup requests.
+Smart Rollups can request arbitrary information through the _reveal data channel_.
+Importantly, as opposed to internal and external messages, the information that passes through the reveal data channel does not pass through layer 1, so it is not limited by the bandwidth of layer 1 and can include large amounts of data.
 
-A rollup can do the following requests through the reveal data channel:
+The reveal data channel supports these requests:
 
-1.  **preimage requests**: The rollup can request arbitrary data of at
-    most 4kBytes, provided that it knows its (blake2b) hash. The request
-    is fulfilled by the rollup node (see [Populating the Reveal Channel](#populating-the-reveal-channel)).
-2.  **metadata requests**: The rollup can request information from the
-    protocol, namely the address and the origination level of the rollup
-    node itself. The rollup node retrieves this information through RPCs
-    to answer the rollup.
+- A rollup node can request an arbitrary data page up to 4KB if it knows the blake2b hash of the page, known as _preimage requests_.
+To transfer more than 4KB of data, rollups must use multiple pages, which may contain hashes that point to other pages.
 
-Information passing through the reveal data channel does not have to be
-considered by layer 1: for this reason, the volume of information is
-not limited by the bandwidth of layer 1. Thus, the reveal data
-channel can be used to upload large volumes of data to the rollup.
+- A rollup node can request information about the rollup, including the address and origination level of the rollup, known as _metadata requests_.
 
-## Origination
+{/*
+TODO how is this data provided?
+Where does it come from?
+Do we need instructions on how to provide data?
+Eventually include:
+  - importing data from a DAC certificate (which can contain anything ultimately, including a kernel to upgrade to)
+  - revealing data from the (WIP) DAL
+*/}
+
+## Smart Rollup lifecycle
+
+The general flow of a Smart Rollup goes through these phases:
+
+1. Origination: A user called the _rollup operator_ originates the Smart Rollup to layer 1 and one or more users start nodes based on that Smart Rollup to independently verify its operation.
+1. Commitment periods: The Smart Rollup nodes receive the messages in the Smart Rollup inbox, run processing based on those messages, generate but do not run outbox messages, and publish a hash of their state at the end of the period.
```

Comment:
As the next point talks about published commitments, you have to introduce it here:
```suggestion
1. Commitment periods: The Smart Rollup nodes receive the messages in the Smart Rollup inbox, run processing based on those messages, generate but do not run outbox messages, and publish a hash of their state at the end of the period., called a commitment
```